misk-hibernate/src/main/kotlin/misk/hibernate/SecretColumn.kt

```diff
@@ -0,0 +1,32 @@
+package misk.hibernate
+
+/**
+ * [SecretColumn] is an annotation used to get Hibernate to encrypt a field before writing it
+ * to the database.
+ * The [keyName] string is used to specify the name of the key to be used to encrypt and decrypt the value.
+ * Install [misk.crypto.CryptoModule] to configure the keys the app uses.
+ * Example:
+ * In app-common.yaml:
+ * ```
+ * crypto:
+ *   keys:
+ *     - key_name: "secretColumnKey"
+ * ```
+ * Then, in an entity class:
+ * ```
+ * @Column
+ * @SecretColumn(keyName = "secretColumnKey")
+ * var secret: String
+ * ```
+ * Hibernate fields annotated with [SecretColumn] must be declared as `VARBINARY()`
```

Comment:
Please call out that these columns can't be part of an index somewhere

Reply:
Good point, I updated the docs